CMakeLists.txt

```diff
@@ -41,13 +41,23 @@ add_library(vkgc_headers INTERFACE)
 
 ### Options that affect the headers ####################################################################################
 option(LLPC_ENABLE_SHADER_CACHE "Enable experimental shader cache" OFF)
+
 if(LLPC_ENABLE_SHADER_CACHE)
-    target_compile_definitions(vkgc_headers INTERFACE LLPC_ENABLE_SHADER_CACHE=1)
+    target_compile_definitions(vkgc_headers INTERFACE LLPC_ENABLE_SHADER_CACHE)
```

Comment:
Why was this changed? IIRC we have some checks that expect this to be either 0 or 1, no? https://github.com/GPUOpen-Drivers/llpc/search?q=LLPC_ENABLE_SHADER_CACHE

Reply:
@Flakebi could you please help comment?

Reply:
As far as I can see, none of the uses checks for `=1`, so I made it consistent with the other flags that are not explicitly set to 1 either.
(The `config.llpc_enable_shader_cache == 'ON'` checks the CMake variable, not the preprocessor definition as far as I understand.)